sourced/ml/utils/engine.py

```diff
@@ -49,10 +57,20 @@ def add_bblfsh_dependencies(bblfsh, config=None):
 
 
 def create_engine(session_name, repositories,
-                  bblfsh=EngineDefault.BBLFSH,
-                  engine=EngineDefault.VERSION,
+                  bblfsh=None,
+                  engine=None,
                   config=None, packages=None, memory="",
                   repository_format="siva", **spark_kwargs):
+    if not bblfsh:
+        bblfsh = "localhost"
+    if not engine:
+        try:
```

Comment:
This is a duplication, please refactor this into a function and use in two places

Reply:
my bad I forgot to delete the class `EngineDefaults`, so this is now only used in `create_engine`. the multiple calls to the APIs were done because in the executors there was imports done to this file, which prompted the class to be redefined each time. since the class is used nowhere else I don't think its useful to keep it - I don't really see a situation where we would need to use it either